### ISSUE TITLE
Can not find ./target/release/capsule_manager

### ISSUE THREAD

**Issue by ClaudiaCumberbatch** [secretflow/capsule-manager#38]
### Issue Type

Bug

### Source

source

### Capsule Manager Version

0.4.0b0

### Capsule Manager SDK Version

0.4.0b0

### Tee Apps Version

0.4.0b0

### OS Platform and Distribution

Ubuntu 20.04.6 LTS

### Python version

Python 3.10.14

### What happend and What you expected to happen.

I followed the main README.md and tried to compile from the source code in simulation mode. But I had trouble finding the compiled binary file. According to [this](https://github.com/asterinas/trustflow-capsule-manager#simulation-mode-1), there should be a build file under ./script/, but I only found build.sh file. So I ran `./script/build.sh -p sim`, and then the target directory was generated, but I was unable to find capsule_manager file under ./target/release/ dir.

### Reproduction code to reproduce the issue.

```shell
N/A
```

**Comment by zimu-yuxi** [comment]
Yes, we have also reproduced the same problem. Let's take a look internally

**Comment by zimu-yuxi** [comment]
Sorry for taking so long to reply. You can refer to [the official website documentation](https://www.secretflow.org.cn/zh-CN/docs/trustflow/0.4.0b0/quick_start/step1). [The current documentation ](https://github.com/asterinas/trustflow-capsule-manager#simulation-mode-1)has not been updated
